### PR TITLE
rename Tab component to Segment

### PR DESCRIPTION
**Pull request by marasva**
## What is the reason for this change?

Tab component is not actually tabs and is thus renamed. The renaming is done due to creation of a new Tab component.

### What is the expected result?

There should not be any changes that are viewable.

### How is it solved?
Check everything looks the same.

### CODE REVIEW

File: packages/ui/src/components/Segment/Segment.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  const [firstTab, setFirstTab] = useState(true)
+  return (
+    <SegmentGroup>
+      <Segment active={firstTab} onClick={() => setFirstTab(true)}>
+        First Segment
+      </Segment>
+      <Segment active={!firstTab} onClick={() => setFirstTab(false)}>
+        Second Segment
+      </Segment>
+    </SegmentGroup>
```
Comment (t0ff3r) [inline packages/ui/src/components/Segment/Segment.stories.tsx:17]: Maybe rename state here aswell?

File: packages/web/src/components/Header/PublicHeader.tsx [hunk 2]
```diff
@@ -396,7 +396,7 @@ export const PublicHeader: React.FC<PublicHeaderProps> = ({
     return (
       <>
         <div className={css.mainPageMobileMenu}>
-          <TabGroup>{customerSegmentTabs}</TabGroup>
+          <SegmentGroup>{customerSegmentTabs}</SegmentGroup>
```
Comment (t0ff3r) [inline packages/web/src/components/Header/PublicHeader.tsx:399]: maybe just customerSegments if it does not affect too much code?

**Review by t0ff3r — COMMENTED** [review]
Looks good. Maybe fix the minor comments and we're good?

File: packages/web/src/components/Header/PublicHeader.tsx (hunk 2 above)
Comment (marasva) [reply packages/web/src/components/Header/PublicHeader.tsx]: yes, fixed

**Review by t0ff3r — APPROVED** [review]

**Pull request merged**